data/src/androidTest/java/com/example/tmdb/data/storages/datastores/EncryptedPrefsDatastoreImplTest.kt

```diff
@@ -0,0 +1,83 @@
+package com.example.tmdb.data.storages.datastores
+
+import android.content.Context
+import androidx.test.core.app.ApplicationProvider
+import app.cash.turbine.test
+import io.kotest.matchers.shouldBe
+import kotlinx.coroutines.test.TestScope
+import kotlinx.coroutines.test.runTest
+import org.junit.Test
+
+class EncryptedPrefsDatastoreImplTest {
+    private val testScope = TestScope()
+    private val context = ApplicationProvider.getApplicationContext<Context>()
+    private val encryptedUserDatastoreImpl = EncryptedPrefsDatastoreImpl(context)
+    private val TOKEN_TYPE = "Bear"
```

Comment:
const value should be declared on the companion scope.

Reply:
Fixed